src/pages/settings/Subscription/CardSection/BillingBanner/BillingBanner.tsx

```diff
@@ -50,12 +53,18 @@ function BillingBanner({title, subtitle, icon, brickRoadIndicator, style, titleS
                 {typeof title === 'string' ? <Text style={[styles.textStrong, titleStyle]}>{title}</Text> : title}
                 {typeof subtitle === 'string' ? <Text style={subtitleStyle}>{subtitle}</Text> : subtitle}
             </View>
-
-            {!!brickRoadIndicator && (
+            {rightIcon ? (
```

Comment:
Maybe we should just pass a component here via props. In my upcoming PR I will be adding a close button here when banner displays a success. Here is a sneakpeak of what I have atm

```            
            {isError && shouldShowRedDotIndicator && (
                <Icon
                    src={Expensicons.DotIndicator}
                    fill={theme.danger}
                />
            )}
            {!isError && shouldShowGreenDotIndicator && (
                <Icon
                    src={Expensicons.DotIndicator}
                    fill={theme.success}
                />
            )}
            {!isError && (
                <PressableWithoutFeedback
                    onPress={() => {
                        Subscription.resetRetryBillingStatus();
                    }}
                    style={[styles.touchableButtonImage]}
                    role={CONST.ROLE.BUTTON}
                    accessibilityLabel={translate('common.close')}
                >
                    <Icon
                        src={Expensicons.Close}
                        fill={theme.icon}
                    />
                </PressableWithoutFeedback>
            )}
```

Reply:
I'll leave it up to you if you want to change it now or leave it to me and I will refactor it on my end in next PR